README.rst

```diff
@@ -78,3 +78,39 @@ Repositories
 .. _`clear-containers-image`: https://download.clearlinux.org/current/
 
 .. _`clear-containers-selinux`: https://github.com/clearcontainers/proxy/tree/master/selinux
+
+Local Building
+==============
+
+Any user can build their own Clear Containers packages using the source files provided in this
+repository, either using the OBS platform or in their own machines using the osc (Open Build Service command line tool).
+
+A Dockerfile is provided to build a container that has all the tools needed to perform local builds.
+
+.. code-block:: shell
+
+   # cd into this repository
+   cd < this repo location>
+
+   # build the container image
+   # cc-osc name is used for the tag, it can be anything.
+   docker build -t cc-osc .
```

Comment:
The `docker build` commands should probably be prepended with `sudo` for maximum portability.